packages/country-config/src/forms.ts

```diff
@@ -239,10 +252,62 @@ export const idVerificationBanner = (
   };
 };
 
-export const idVerificationFields = (event: string, sectionId: string, mapping: any) => {
+export const getInitialValueFromIDReader = (fieldNameInReader: string) => ({
+  dependsOn: ["idReader", "esignetCallback"],
+  expression: `$form?.idReader?.${fieldNameInReader} || $form?.esignetCallback?.data?.${fieldNameInReader} || ""`,
+});
+
+export const idReaderFields = (
+  event: "birth" | "death",
+  section: "informant" | "mother" | "father",
+  qrConfig: QRConfig,
+  esignetConfig: ESignetConfig | undefined,
+  verifiedCustomFieldMapping: any,
+  conditionals: any[] = [],
+) => {
+  const readers: any[] = [qr(qrConfig)];
+  const fields: any[] = [
+    idReader(
+      event,
+      section,
+      conditionals.concat({
+        action: "hide",
+        expression:
+          "$form?.verified === 'verified' || $form?.verified === 'authenticated' || $form?.verified === 'failed'",
```

Comment:
there is no 'authenticated' status handling in core yet, will add the authenticated status in my next batch of work